### PR TITLE
Do not use curl but copy requirements.txt

### PR DESCRIPTION
**Pull request by pacospace**
## Related Issues and Dependencies

See issue: https://github.com/sclorg/s2i-python-container/issues/466#issuecomment-888949810

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

COPY requirements txt instead of using curl

### CODE REVIEW

File: f34-py39/Dockerfile [hunk 1]
```diff
@@ -26,13 +26,13 @@ LABEL summary="$SUMMARY" \
 
 USER 0
 COPY ./s2i_assemble.patch /tmp/s2i_assemble.patch
+COPY ./requirements.txt /opt/app-root/src/requirements.txt
```
Comment (frenzymadness) [inline f34-py39/Dockerfile:29]: There is an env variable for the path to the source code you might want to use.

```suggestion
COPY ./requirements.txt $HOME/requirements.txt
```

Comment (pacospace) [reply f34-py39/Dockerfile]: Thanks!! Adjusted in https://github.com/thoth-station/s2i-thoth/pull/210/commits/0e4cc839938df4bcae82aa79598657bd6162596e

**Review by harshad16 — APPROVED** [review]
lgtm
thanks :100:

**Pull request merged**